tests/acceptance/features/bootstrap/ImpersonateAppContext.php

```diff
@@ -126,14 +126,15 @@ public function createUserWithGroup(string $username, string $password, string $
 	}
 
 	/**
-	 * @Given /^the administrator has created following users:$/
+	 * @Given /^the administrator has created following users with group and role:$/
```

Comment:
role is optional here :thinking: maybe
```suggestion
	 * @Given /^the administrator has created following users with settings:$/
```